app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt

```diff
@@ -809,7 +808,7 @@ class StateFragmentTest {
   }
 
   @Test
-  @Ignore("Currently failing due to a regression") // TODO(#1769): Re-enable.
+  @RunOn(TestPlatform.ESPRESSO) // TODO(#1612): Enable for Robolectric.
```

Comment:
Note there are identical TODOs for adjacent tests, portrait versions of these two.